felix/proto/felixbackend.proto

```diff
@@ -112,18 +112,23 @@ message ToDataplane {
     // VXLANTunnelEndpointRemove is send when a route is deleted.
     VXLANTunnelEndpointRemove vtep_remove = 26;
 
-    // WireguardEndpointUpdate is sent to setup wireguard on the host.
+    // WireguardEndpointUpdate is sent to setup IPv4 wireguard on the host.
     WireguardEndpointUpdate wireguard_endpoint_update = 27;
-    // WireguardEndpointRemove is sent to undo wireguard on the host.
+    // WireguardEndpointRemove is sent to undo IPv4 wireguard on the host.
     WireguardEndpointRemove wireguard_endpoint_remove = 28;
 
+    // WireguardEndpointV6Update is sent to setup IPv6 wireguard on the host.
```

Comment:
We shouldn't change field numbers for existing fields once they are in use, since it means that older clients will no longer work with new clients (these numberical identifiers are used to identify the fields once serialized).

Reply:
Sorry about that, will fix